README.md

```diff
@@ -1,71 +1,95 @@
-- download songs via yt-dlp
-- KISS (keep it simple and stupid) front end
-- MPD integration (use it on a Raspberry Pi!)
+- download songs, albums, public playlists
+- automatically generated m3u8 playlist
+- download as zip
+- ~~320 kbit/s~~ (currently only 128 kbit/s, see [issue #66](https://github.com/kmille/deezr-downloader/issues/66#issuecomment-2002309521))
+- ID3 tags and album cover
+- [KISS](https://en.wikipedia.org/wiki/KISS_principle) frontend
 - simple REST api
-- proxy support (https/socks5)
-
-
-### How to use it
+- proxy support: https/socks5
+- MPD integration: use it on a Raspberry Pi!
+
+## Table of Contents:
+- [Get started](#get-started)
+  - [Install Python](#1-install-python)
+  - [Install deezer-downloader](#2-install-deezer-downloader)
+  - [Retrieve your arl cookie](#3-retrieve-your-arl-cookie)
+  - [Set the config file](#4-set-the-config-file)
+  - [Run deezer-downloader](#5-run-deezer-downloader)
+  - [Access the frontend](#6-access-the-frontend)
+- [Specific use cases](#specific-use-cases)
+  - [Run with Docker](#run-with-docker)
+  - [Run with Vagrant](#run-with-vagrant)
+  - [Run as a service](#run-as-a-service)
+  - [Developer setup](#developer-setup)
+  - [Deployment with Ansible](#deployment-with-ansible)
+- [Settings](#settings)
+- [Screenshots](#screenshots)
+- [Tests](#tests)
+- [Changelog](#changelog)
+
+## Get started
+
+### 1. Install Python
+[Here.](https://www.python.org/about/)
+
+### 2. Install deezer-downloader
+```bash
+pip install --user deezer-downloader
+```
 
-There is a settings file template called `settings.ini.example`. You can specify the download directory with  `download_dir`. Pressing the download button only downloads the song/album/playlist. If you set `use_mpd=True` in the `settings.ini` the backend will connect to mpd (localhost:6600) and update the music database. Pressing the play button will download the music. If `use_mpd=True`  is set the mpd database will be updated and the song/album/playlist will be added to the playlist. In `settings.ini` `music_dir` should be the music root location of mpd. The `download_dir` must be a subdirectory of `music_dir`. 
+### 3. Retrieve your `arl` cookie
+On Firefox or Chrome-based browser:
+- Log into your **FREE** Deezer account (do **NOT** use a premium account)
+- Open the DevTools (`F12` or `Ctrl+Shift+C` or `Ctrl+Shift+I`)
+- Go to `Storage` tab
+- In the cookies, find `arl`: a ~200 characters alphanumeric key
+
+### 4. Set the config file
+Retrieve the template: 
+```bash 
+deezer-downloader -t
+```
+You need to set at least:
+- under `[deezer]`: `cookie_arl`, your **NON PREMIUM** arl key
+- under `[youtubedl]`: `command`, your yt-dlp install path\
+As stated in the config template, deezer-downloader do NOT keep yt-dlp updated, you will have to monitor this yourself.
```

Comment:
I suggest to change the sentence as follows : 

As stated in the config template, deezer-downloader does NOT keep `yt-dlp` updated for you; updating `yt-dlp` is your responsibility.

Reply:
Yes, if it suits @kmille I'm okay with your modification.